tests/lib/spread/rules/main.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+  tests:
+    from:
+      - tests/main/.*
+      - tests/core/.*
+      - tests/completion/.*
+      - tests/cross/.*
+      - tests/regression/.*
+      - tests/smoke/.
+      - tests/unit/.*
+      - tests/upgrade/.*
+    to: [$SELF]
+
+  unit:
+    from: [.*_test.go]
```

Comment:
We have some test-only packages like seedtest, not sure if they need to be considered here or not.

Reply:
we could consider those files in the future, I would start easy and then go a bit more on details. The idea of this change is to monitor how the tool works before we move to this solution